.github/workflows/helm_pr_deployment.yml

```diff
@@ -0,0 +1,93 @@
+name: Deploy PR to Kubernetes
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  deploy:
+    if: ${{ github.event.pull_request.head.repo.full_name == 'ls1intum/Artemis' }}
```

Comment:
So this will be executed for every commit in nearly every PR from now on. This would lead to some useless spam in the activity log of every PR.
Ideally, you would check for some label to be present before requesting to deploy this to an environment.

Reply:
Created `deploy:k8s`